### ISSUE TITLE
`require()` issue

### ISSUE THREAD

**Issue by crobinson42** [panosoft/node-chartist#5]
Can anyone explain this?

```
> require('node-chartist')
Error: ENOENT: no such file or directory, open '/Users/willlyrob/projects/therms/node_modules/node-chartist/node_modules/chartist/dist/chartist.js'
    at Error (native)
    at Object.fs.openSync (fs.js:549:18)
    at Object.fs.readFileSync (fs.js:397:15)
    at Object.<anonymous> (/Users/willlyrob/projects/therms/node_modules/node-chartist/lib/chart.js:14:26)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
```

**Comment by crobinson42** [comment]
This is the culprit: https://github.com/panosoft/node-chartist/blob/master/lib/chart.js#L14

**Comment by alexgig** [comment]
Hello @crobinson42,

Thank you for posting this issue. This is most likely related to the following: https://github.com/panosoft/node-chartist/issues/2#issuecomment-174591453

Please feel free to reopen this issue if that does not resolve it for you.

Best,
Alex

**Comment by ScheepersDeBruin** [comment]
In other words, the "culprit" should be:
var chartistSource =  fs.readFileSync(path.resolve(__dirname, '../../chartist/dist/chartist.js'));
in stead of:
var chartistSource =  fs.readFileSync(path.resolve(__dirname, '../node_modules/chartist/dist/chartist.js'));

**Comment by alexgig** [comment]
Hello all,

This has been resolved in `+ node-chartist@0.2.8`

Best,
Alex
